messaging/service-worker.js

```diff
@@ -38,16 +40,18 @@ function onBackgroundMessage() {
 
   // [START messaging_on_background_message]
   messaging.onBackgroundMessage((payload) => {
-    console.log('[firebase-messaging-sw.js] Received background message ', payload);
+    console.log(
+      '[firebase-messaging-sw.js] Received background message ',
+      payload
+    );
     // Customize notification here
     const notificationTitle = 'Background Message Title';
     const notificationOptions = {
       body: 'Background Message body.',
-      icon: '/firebase-logo.png'
+      icon: '/firebase-logo.png',
```

Comment:
Let's avoid trailing commas - I'll be sure to add it to prettier.
```suggestion
      icon: '/firebase-logo.png'
```